Setup/Patch/Data/ImageRoles.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Macopedia\Allegro\Setup\Patch\Data;
+
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+use Magento\Framework\Setup\Patch\PatchInterface;
+use Magento\Eav\Setup\EavSetup;
+use Magento\Eav\Setup\EavSetupFactory;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+
+class ImageRoles implements DataPatchInterface
```

Comment:
Please add description what this class is responsible for.

Reply:
@tmotyl Done.